src/containers/ResetPassword/__snapshots__/ResetPassword.test.js.snap

```diff
@@ -0,0 +1,99 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I believe this file should be ignored in git.

Reply:
the same thing I told to @aashrafh  "I really do it, so when I interact with api, I need to spy on the both functions of button and form, so shallow the TextField gonna save much headache"" @AbdullahMuhammed5